docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -27,9 +24,6 @@ This method requires as parameters:
 Disable statistics DataWriters
 ------------------------------
 
-.. warning::
-    This method will be implemented in future releases. For the moment it always returns ``RETCODE_UNSUPPORTED``.
-
```

Comment:
Missing snippet

Reply:
I have added a complete code snippet using all the DDS API after explaining the API